js/autotrack/reactNavigation.js

```diff
@@ -78,14 +110,33 @@ export const withReactNavigationAutotrack = track => AppContainer => {
                 'Heap: React Navigation is instrumented for autocapture.'
               );
               this.topLevelNavigator = navigatorRef;
```

Comment:
i'm wondering why use class variables for this instead of the class's `state`? I'm not sure if there's differences between RN and React in terms of class state and render lifecycles so genuinely curious

Reply:
AFAIK we don't want to update `state` (thus triggering a re-render) every time the ref changes